docs/installguide/system_requirements.rst

```diff
@@ -1,12 +1,87 @@
 System requirements
 ===================
 
+Operating systems
+-----------------
+
+ * Windows XP, Vista, 7, 8, 10
```

Comment:
Yes I think we do, but I'll move it to a separate list of limited support! It does have issues, especially on the browser side.